app/models/ability.rb

```diff
@@ -0,0 +1,28 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+      if user.has_role?(:admin)
+        can :manage, :all
+      else
+        can :read, :all
```

Comment:
When we introduce more roles like `author`, `reader` this will have to be changed. Please introduce roles `author` and `reader`.

1. `author` can create articles and update, delete only their own articles which they have created.
2. `reader` can only read articles and cannot modify or delete any.